Lib/traceback.py

```diff
@@ -1732,6 +1715,38 @@ def _compute_suggestion_error(exc_value, tb, wrong_name):
     return suggestion
 
 
+def _suggestion_message(exc_type, exc_value, exc_traceback):
+    if (
+        issubclass(exc_type, ModuleNotFoundError)
+        and sys.flags.no_site
+        and getattr(exc_value, "name", None) not in sys.stdlib_module_names
+    ):
+        return ("Site initialization is disabled, did you forget to "
+            "add the site-packages directory to sys.path?")
+    if issubclass(exc_type, (ImportError, NameError, AttributeError)):
+        if issubclass(exc_type, ImportError):
+            wrong_name = getattr(exc_value, "name_from", None)
+        else:
+            wrong_name = getattr(exc_value, "name", None)
+        if wrong_name:
+            other_name = _compute_suggestion_error(
+                exc_value, exc_traceback, wrong_name
+            )
+            maybe_builtin_import = (
```

Comment:
I just remembered that, in addition to `sys.stdlib_module_names`, we also have `sys.builtin_module_names`, so let's rename this variable `maybe_stdlib_import` instead of `maybe_builtin_import`.